man/chpl.rst

```diff
@@ -424,6 +424,25 @@ OPTIONS
     example, they might be able to hoist a 'get' out of a loop. See
     $CHPL\_HOME/doc/rst/technotes/llvm.rst for details.
 
+**--llvm-print-ir <name>**
+    Print intermediate representation (IR) of function named <name>. 
+    Need to specify stage using  **--llvm-print-ir-stage** in order 
+    to be printed.
+
+**--llvm-print-ir-stage <stage>**
+    Picks stage from which to print LLVM IR of function defined in 
+    **--llvm-print-ir**. 
+    Chapel compiler runs many different optimization passes each of which
+    can change IR of function. This option allows to pick IR of function
```

Comment:
"can change IR of function" -> "can change the IR of functions"

"allows to pick IR" -> "allows one to pick the IR"